sources/web/datalab/polymer/components/resizable-divider/resizable-divider.ts

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+/**
+ * Resizable divider element for Datalab.
+ * This element is a container for two elements with a vertical resize divider.
+ */
+class ResizableDividerElement extends Polymer.Element {
+
+  /**
+   * Minimum pane width in pixels. Defaults to 50(px);
+   */
+  public minimumWidthPx: number;
+
+  /**
+   * Initial position of the divider in percentage. Defaults to 50(%);
+   */
+  public initDividerPosition: number;
+
+  private _boundMouseDownHandler: EventListenerOrEventListenerObject;
+  private _boundMouseupHandler: EventListenerOrEventListenerObject;
+  private _boundMouseMoveHandler: EventListenerOrEventListenerObject;
+  static get is() { return 'resizable-divider'; }
+
+  static get properties() {
+    return {
+      initDividerPosition: {
+        type: Number,
+        value: 50,
+      },
+      minimumWidthPx: {
```

Comment:
Well what about `pt`, `em`, or `vw`? It's a lot of work to support the entire list, I think we should stick with what fulfills our requirements for now, and add support for things as we go.

The way I see it is you need to be able to specify the size of an element in percent points such that it scales up and down when the window resizes, but also its minimum size in pixels for when you have elements inside it that you don't want to get messed up. I think these two are most important for usability of the element, but if you have a use case in mind that you think we should support we can discuss it.